service/routes.py

```diff
@@ -5,10 +5,9 @@
 
 The service has the 6 following routes: Create, Read, Update, Delete, List and the root.
 """
-import json
-from flask import Flask, jsonify, request, url_for, make_response, abort
```

Comment:
And I used abort in "read_promotions". Maybe you should start linting after merging my pull request.